subsys/net/l2/ppp/lcp.c

```diff
@@ -206,6 +206,11 @@ static void lcp_finished(struct ppp_fsm *fsm)
 					       lcp.fsm);
```

Comment:
Minor nit about the commit subject, no need to mention literal `subsys` there,
just do `net: ppp: Properly .....`